api/database/FormsDOM.py

```diff
@@ -4,5 +4,41 @@
 
 app = Flask(__name__)
 MONGO_URL = os.environ.get('MONGODB_URI')
-app.config["MONGO_URI"] = MONGO_URL
+app.config["MONGO_URI"] = "mongodb://localhost:27017/sfja"
 mongo = PyMongo(app)
+
+# Creates new form.
```

Comment:
This is a nit pick, but if your function name is clear enough in what it does, you don't need a comment that restates it.